gradle/verification-metadata.xml

```diff
@@ -2322,6 +2322,11 @@
             <sha256 value="b35af3a5ffb1107e5e2d0b1a1c0ec8c5198c0691ff03600346efc439c632b63b" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.1.7">
```

Comment:
Can you only keep the entry for the last version of the retry plugin?

Reply:
removed - sorry I assumed wrongly old versions should stay in here.